### ISSUE TITLE
[Toggler] aria-expanded value out of sync if target element initial state is already expanded/visible

### ISSUE THREAD

**Issue by felipe-spengler-osf** [zurb/foundation-sites#11785]
## What should happen?
`aria-expanded` on toggler link should correctly report true/false based if the target element is visible or not.
...

## What happens instead?
If the target element default state is already visible/expanded (as opposed to all the examples on DOCs), while the aria-expanded attribute on the toggler link open ups correctly set to "true", on the first time we click on it (in this case to hide the target element), aria-expanded doesn't get updated to "false", it only starts updating after you click again, therefore it will be out of-sync.

...

## Possible Solution
I believe the function is assuming the target element is always initially hidden, which is incorrect, so the first click is just always setting aria-expanded to true. It should verify the target element visible state or have some other option to inform toggler the initial state of the target element.

...

## Test Case and/or Steps to Reproduce (for bugs)
You can reproduce this with Foundation 6.5.3 here: https://codepen.io/anon/pen/pXqNaO

**Comment by felipe-spengler-osf** [comment]
Update: further investigation shows that apparently the root of the issue is that toggler work based on the premise that the specified class should be the one to make the element visible.

So instead of using "hide", the target element should have a "is-visible" class on the data-toggler and if want the element already visible as initial state, make it already have the is-visible class.

This way aria-expanded will work as expected.

If this is intended then maybe a note/update should be done on the DOC's explaining how to proper setup if want the element to be expanded by default.

**Comment by DanielRuf** [comment]
Or simply check `.is(':visible')`.

**Comment by scottsawyer** [comment]
I agree, toggler should check .is(':visible') when setting the aria-expanded.  I have a situation where it's not desirable for the item to be visible for mobile, but should be toggled, basically a collapsible menu, but only for mobile.  I am using the class 'hide-for-small-only' on the menu wrapper, and toggling that class displays the menu perfectly, but the aria-expanded is always out of sync.  

Initial page load, menu hidden - aria-expanded="false" ( this is correct ) 
First click on the trigger, menu displayed - aria-expanded="false" ( this is incorrect )
Second click on the trigger, menu hidden - aria-expanded="true" ( this is incorrect )

From here on, the aria-expanded attribute is always incorrect.

I haven't tested, but I imagine it is a problem using any of the .hide-for-x classes to toggle visibility.

https://codepen.io/scottsawyer/pen/NWPbMeM
